feature/onboarding/src/main/java/com/mashup/dorabangs/feature/onboarding/OnBoardingRoute.kt

```diff
@@ -0,0 +1,32 @@
+package com.mashup.dorabangs.feature.onboarding
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.orbitmvi.orbit.compose.collectAsState
+import org.orbitmvi.orbit.compose.collectSideEffect
+
+@Composable
+fun OnBoardingRoute(
+    modifier: Modifier = Modifier,
+    viewModel: OnBoardingViewModel = hiltViewModel(),
+    navigateToHome: () -> Unit = {},
+) {
+    val state by viewModel.collectAsState()
+
+    viewModel.collectSideEffect { sideEffect ->
+        when (sideEffect) {
+            OnBoardingSideEffect.NavigateToHome -> {
+                navigateToHome.invoke()
+            }
+        }
+    }
+
+    OnBoardingScreen(
+        modifier = modifier,
+        state = state,
+        onClickKeyword = { index -> viewModel.onClickKeyword(index) },
```

Comment:
이거만 밑에랑 다른 이유는 무엇 인 . 가용. ?
```suggestion
        onClickKeyword = viewModel::onClickKeyword,
```

Reply:
Int땜에 그런 것 같운뎅 아마 자동으루 들어갈거야요

Reply:
@Ahn-seokjoo 
감사합니다 선생님 수정할게요